opensearch-observability/src/main/kotlin/org/opensearch/observability/index/ObservabilityIndex.kt

```diff
@@ -58,6 +58,7 @@ internal object ObservabilityIndex {
 
     private lateinit var client: Client
     private lateinit var clusterService: ClusterService
+    private var mappingsUpdated: Boolean = false
```

Comment:
could you move this above the lateinit ones

Reply:
Is there just a stylistic choice? Or is there another reason?

Reply:
just code style, to group non-lateinit variables together